taskcluster/ci/config.yml

```diff
@@ -157,20 +158,43 @@ release-promotion:
         promote:
             target-tasks-method: promote_xpi
     notifications:
-        # configure privileged webextension email addresses for
+        # configure mozillaonline-privileged webextension email addresses for
         # notifications
+        mozillaonline-privileged:
+            by-phase:
+                build:
+                    - "aki+mozillaonline-priv-webext-build@mozilla.com"
+                    - "hectorz+mozillaonline-priv-webext-build@mozilla.com"
+                    - "mkaply+mozillaonline-priv-webext-build@mozilla.com"
+                    - "awagner+mozillaonline-priv-webext-build@mozilla.com"
+                promote:
+                    - "aki+mozillaonline-priv-webext-promote@mozilla.com"
+                    - "hectorz+mozillaonline-priv-webext-promote@mozilla.com"
+                    - "mkaply+mozillaonline-priv-webext-promote@mozilla.com"
+                    - "awagner+mozillaonline-priv-webext-promote@mozilla.com"
+        # configure normandy-privileged webextension email addresses for
+        # notifications
+        normandy-privileged:
+            by-phase:
+                build:
+                    - "aki+normandy-priv-webext-build@mozilla.com"
+                    - "rdalal+normandy-priv-webext-build@mozilla.com"
+                    - "mcooper+normandy-priv-webext-build@mozilla.com"
+                promote:
+                    - "aki+normandy-priv-webext-promote@mozilla.com"
+                    - "rdalal+normandy-priv-webext-promote@mozilla.com"
+                    - "mcooper+normandy-priv-webext-promote@mozilla.com"
+        # configure privileged webextension email addresses for notifications
         privileged:
             by-phase:
                 build:
                     - "aki+priv-webext-build@mozilla.com"
-                    - "rdalal+priv-webext-build@mozilla.com"
                     - "awagner+priv-webext-build@mozilla.com"
-                    - "mcooper+priv-webext-build@mozilla.com"
+                    - "pkewisch+priv-webext-build@mozilla.com"
```

Comment:
if there's a mailing list we can use for these, that would be preferred. Not a blocker, but something we'll want to fix soonish.

Reply:
f0d6920.

Reply:
Great! This mailing list exists already?

Reply:
It does.